packages/api-client-core/src/support.ts

```diff
@@ -543,3 +545,87 @@ const checkEquality = (a: any, b: any, refs: any[]): boolean => {
 };
 
 export const isEqual = (a: any, b: any) => checkEquality(a, b, []);
+
+/**
+ * Processes the flexible, convenient JS-land inputs for an action to the fully qualified GraphQL API inputs
+ * @internal
+ **/
+export const disambiguateActionVariables = (action: AnyActionFunction, variables: Record<string, any> | undefined) => {
```

Comment:
nit: `normalize` feels more idiomatic (this also applies to `disambiguateBulkActionVariables` below)

```suggestion
export const normalizeActionVariables = (action: AnyActionFunction, variables: Record<string, any> | undefined) => {
```